src/pipes/zod/zod-validation.exception.spec.ts

```diff
@@ -26,13 +26,13 @@ describe('ZodValidationException', () => {
 
     const error = new ZodValidationException(result.error);
 
-    expect(error).toBeInstanceOf(BadRequestException);
+    expect(error).toBeInstanceOf(ZodValidationException);
```

Comment:
Changes _from_ `BadRequestException` to `ZodValidationException`/`HttpException` in this PR fix errors found by the test adjustments.